website/templates/include/profile/settings_navpanel.mako

```diff
@@ -13,5 +13,10 @@
 
       <li class="${'active' if current_page == 'notifications' else ''}">
         <a href="${ '#' if current_page == 'notifications' else web_url_for('user_notifications') }">Notifications</a></li>
+
+      % if dev_mode:
```

Comment:
That's essentially what I'm looking for with this; only show them right now on a staging or local environment. The advantage of making a specific flag is that we'd be able to keep it from showing up on staging while still showing up on staging2, which is fine. The downside is it's another flag. One that will probably be going away soon.

I believe we had this discussion https://github.com/CenterForOpenScience/osf.io/pull/3082/files#diff-2a345ff09b5f57b5b1503ff0fa2060f3R353 and I'm happy to have the same solution for both.